projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-abstract-token-subscription-service.php

```diff
@@ -448,6 +469,22 @@ private function set_token_cookie( $token ) {
 		}
 	}
 
+	/**
+	 * Clear the auth cookie.
+	 *
+	 * @return void
+	 */
+	private function clear_token_cookie() {
+		if ( defined( 'TESTING_IN_JETPACK' ) && TESTING_IN_JETPACK ) {
+			return;
+		}
+
+		if ( isset( $_COOKIE[ self::JWT_AUTH_TOKEN_COOKIE_NAME ] ) && false === headers_sent() ) {
+			unset( $_COOKIE[ self::JWT_AUTH_TOKEN_COOKIE_NAME ] ); // In case cookie gets checked later in code
+			setcookie( self::JWT_AUTH_TOKEN_COOKIE_NAME, '', 1, '/', COOKIE_DOMAIN, is_ssl(), true ); // httponly -- used by visitor_can_view_content() within the PHP context.
```

Comment:
I'm not seeing this work as expected on simple sites with a *.wordpress.com address. It appears `COOKIE_DOMAIN` is detecting `.wordpress.com`, but the cookie domain in my Simple test site is `.chriswsmith9.wordpress.com`.

Reply:
It doesn't work for me for the same reason.

Reply:
We do not rely on the cookie on `*.wordpress.com` address...